python/pyspark/sql/functions.py

```diff
@@ -1028,20 +1028,29 @@ def to_timestamp(col, format=None):
 
 
 @since(1.5)
-def trunc(date, format):
+def trunc(data, truncParam):
```

Comment:
I believe this definitely breaks backward compatibility for keyword-argument usage in Python.